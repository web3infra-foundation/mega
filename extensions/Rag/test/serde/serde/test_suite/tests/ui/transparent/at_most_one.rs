use serde_derive::Serialize;

#[derive(Serialize)]
#[serde(transparent)]
struct S {
    a: u8,
    b: u8,
}

fn main() {}
