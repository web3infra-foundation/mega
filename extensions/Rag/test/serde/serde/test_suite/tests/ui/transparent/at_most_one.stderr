error: #[serde(transparent)] requires struct to have at most one transparent field
 --> tests/ui/transparent/at_most_one.rs:4:1
  |
4 | / #[serde(transparent)]
5 | | struct S {
6 | |     a: u8,
7 | |     b: u8,
8 | | }
  | |_^
